front/.eslintrc.json

```diff
@@ -5,6 +5,12 @@
     "react-hooks/rules-of-hooks": 0,
     "react-hooks/exhaustive-deps": 0,
     "@next/next/no-img-element": 0,
-    "jsx-a11y/alt-text": 0
+    "jsx-a11y/alt-text": 0,
+    "no-undef": 1
```

Comment:
Usually, Typescript projects do not enable this rule as it's already handled by TS

Next is disabling the option too
https://github.com/vercel/next.js/blob/canary/.eslintrc.json#L60

Are you sure you need it?

Reply:
If we are committed to write only TS going forward then you are right, we don't need this 👍